apps/etterlatte-saksbehandling-ui/client/src/components/generellbehandling/Utland.tsx

```diff
@@ -230,6 +252,30 @@ const Utland = (props: { utlandsBehandling: Generellbehandling & { innhold: Utla
                   </>
                 )
               )}
+              {utlandsBehandling.tilknyttetBehandling ? null : (
+                <Alert variant="warning">
+                  Denne utlandsbehandlingen er ikke tilknyttet en behandling. Vi kan derfor ikke hente avdoedes
+                  informasjon
+                </Alert>
+              )}
+              <div>
+                {isSuccess(avdoedeStatus) && avdoed && (
+                  <>
+                    <h3>Informasjon om avdøde</h3>
+                    <p>
+                      <b>Navn</b>
+                      <br />
+                      {formaterNavn(avdoed.opplysning)}
+                      <br />
+                      <b>Fødselsnummer</b>
+                      <br />
+                      {avdoed.opplysning.foedselsnummer}
+                      <br />
+                    </p>
+                  </>
+                )}
+                {isFailure(avdoedeStatus) && <ApiErrorAlert>Klarte ikke å hente informasjon om avdøed</ApiErrorAlert>}
+              </div>
```

Comment:
Ville kanskje `<InfoWrapper>` og `<Info>` gjort samme nytten?

Reply:
Vi har to komponenter som heter InfoWrapper 🤠